Edited content for figure 3 (#22)
greenelab/iscb-diversity-manuscript: content/40.resource-availability.md

A manubot instance that hosts version history for this manuscript is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity-manuscript>.
Our analysis of authors and ISCB-associated honorees is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.

> Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code also distributed under a BSD 3-Clause License.

Reviewer: ```suggestion
Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.
```

Reviewer 2: Note that this is not currently the case. Issue still open at https://github.com/greenelab/wiki-nationality-estimate/issues/3

A manubot instance that hosts version history for this manuscript is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity-manuscript>.
Our analysis of authors and ISCB-associated honorees is available under the Creative Commons Attribution License at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.

> Our Wikipedia name dataset is available under the Creative Commons Attribution License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.

Reviewer 2: ```suggestion
Our Wikipedia name dataset is dedicated to the public domain under CC0 License at <https://github.com/greenelab/wiki-nationality-estimate>, with source code to construct the dataset available under a BSD 3-Clause License.
```
